README.md

```diff
@@ -19,3 +19,91 @@ quantitatively reason with them.
+We must still consider that as the array is randomly sorted, each of these chosen elements (Index 0, middle positon via length of array / 2 rounded if needed, and final position at index arrayLength -1) has an equal chance to be anywhere within the array numerically.
+
+
+Mathmatically, this can be seen via considering the possible outcomes of each of the three selected values being within or otuside of the desired n/2 center range.
+Take, for instance, if all three (position 0, middle position, end position) are all outside of the desired "good" pivot range. 
+Then the selected pivot from the median would be forced to be out of the good range.
+
+So the visualized permutations of the randomly selected three positions would be: 
+
+GGG
+
+GGB GBG BGG 
+
+BBG GBB BGB
+
+BBB
+
+With "G" signifying that a value is within the "good" center range for pivots, and "B" meaning a bad pivot outside of the center range.
+All permutations sharing a row above are in essence the same, as the median pivot selector is assumed to sort the three values taken from the three outlined positions, as they are from a presumably unsorted list and thus have an equal likelihood to be any value.
+
+The easiest to consider are the GGG and BBB, corresponding to all three selected indexes holding values that are either all within or outside of the desired range, respectively.
+
+Next, we can consider the permutations with two elements within the n/2 range. 
+This includes GGB GBG and BGG.
+All of these are forced to sort into BGG or GGB, each of which contain a "G" pivot as desired in the median position.
+
+The only permutations that we have not yet considered are BBG, GBB, and BGB.
+These can be sorted into a variety of results. 
+BBG, GBB, and BGB can all sort into either their existing order or the other orders (at equal probability, as each element is of random numerical value).]
+
+This can be understood as the "B" elements can be either in the 0 to n/4, or 3n/4 to 1 segments.
+They can both be lower than the good pivot, or they can be both higher than the good pivot in numerical value.
+In each of these scenarios, a bad pivot would ultimately be selected as the median.
+However, the third possibility is that one bad pivot is less than the good pivot, and one is greater than the good pivot.
+This would force the good pivot into the median position, resulting in a good pivot median being selected.
+
+Overall, we can now consider the number of permutations that result in a good median pivot, divided by the overall number of permutations.
+
+There are four permutations of the randomly selected elements (again - position 0, middle position, last position) which result in a guaranteed good median pivot being selected.
+There is one permutation that results in a guaranteed bad median pivot being selected.
+There are three permutations that have a 1/3 possibility of being sorted numerically into an order that results in a good pivot being selected.
+
+(4 + 3*(1/3)) / 7 = 5/7
```

Comment:
Not quite following you here -- there are not only 7 different ways of choosing 3 elements.